lang/LangSource/SC_LanguageClient.cpp

```diff
@@ -115,23 +115,6 @@ void SC_LanguageClient::initRuntime(const Options& opt)
 {
 	// start virtual machine
 	if (!mHiddenClient->mRunning) {
-#ifdef __linux__
-		using DirName = SC_Filesystem::DirName;
-		namespace bfs = boost::filesystem;
-		bfs::path deprecatedSupportDirectory = SC_Filesystem::instance().getDirectory(DirName::UserHome);
-		deprecatedSupportDirectory /= "share/SuperCollider";
-
-		if (bfs::exists(deprecatedSupportDirectory)) {
-			bfs::path supportDirectory = SC_Filesystem::instance().getDirectory(DirName::UserAppSupport);
-			postfl("WARNING: Deprecated support directory detected: %s\n"
-				"Extensions and other contents in this directory will not be available until you move them to the new support directory:\n"
-				"%s\n"
-				"Quarks will need to be reinstalled due to broken symbolic links.\n\n",
-				deprecatedSupportDirectory.string().c_str(), // we can safely convert to c_str here because the POSIX filesystem uses UTF-8
-				supportDirectory.string().c_str());
-		}
-#endif
-
```

Comment:
thanks! Can you remove the includes above for boost/filesystem/path, boost/filesystem/operations, and SC_Filesystem? This is the only place they're needed in this file (I guess I should have put SC_Filesystem in the ifdef when I last edited this)

Reply:
Good catch!  Done.